### ISSUE TITLE
Run time generated suites

### ISSUE THREAD

**Issue by chokaa** [jasmine/jasmine#1182]
Hi, I want to run tests when some event occurs in my app/browser, I embedded jasmine inside some app and tried something more complex but i felt like Jasmine doesn't support run time generated tests so i went and write something simple, like : 
describe("Refresh", function () {
        window.addEventListener("click",function(){
            describe("test",function(){
                it("test",function(){
                   expect(true).toBe(true);
                });
                console.log("Some text")
            });
        })
});
So after this i get "Some test" in console but no specs, however if I do this :
describe("Refresh", function () {

            describe("test",function(){
                it("test",function(){
                   expect(true).toBe(true);
                });
                console.log("Some text")
            });
        
});
Basically same thing just without window,addEventListener it work. Is there any way i could do this kind of testing in Jasmine and if not can you suggest some framework that supports this kind of testing.
Best regards, Nikola Ilic

**Comment by slackersoft** [comment]
You are more likely to get quicker responses from the community for "How to use jasmine?" questions and a history of other solutions on the [jasmine-js group](https://groups.google.com/group/jasmine-js). We try to keep jasmine's github issues list focused on bugs and feature requests for jasmine itself.

Closing this. Thanks for using Jasmine!
